src/Tribe/Main.php

```diff
@@ -2058,6 +2058,7 @@ public function localizeAdmin() {
 				'prevText'        => esc_html__( 'Prev', 'the-events-calendar' ),
 				'currentText'     => esc_html__( 'Today', 'the-events-calendar' ),
 				'closeText'       => esc_html__( 'Done', 'the-events-calendar' ),
+				'ajaxurl'         => esc_url_raw( admin_url( 'admin-ajax.php', ( is_ssl() ? 'https' : 'http' ) ) ),
```

Comment:
Just musing, but would there be a problem if `false === is_ssl()` (frontend) but `defined( 'FORCE_SSL_ADMIN', true )`?